dblookupext/nilHistoryRepository.go

```diff
@@ -36,6 +36,11 @@ func (nhr *nilHistoryRepository) IsEnabled() bool {
 	return false
 }
 
+// RevertBlock -
+func (nhr *nilHistoryRepository) RevertBlock(_ data.HeaderHandler, _ data.BodyHandler) error {
+	return nil
```

Comment:
either add tests for this or move the implementation in a `disabled` package

Reply:
moved.